src/peripheral/mod.rs

```diff
@@ -130,6 +131,9 @@ pub struct Peripherals {
     /// Nested Vector Interrupt Controller
     pub NVIC: NVIC,
 
+    /// Security Attribution Unit
+    pub SAU: SAU,
```

Comment:
I think this should be `cfg`d out when not targeting armv8, and the doc comment should say that it only exists then.

I know we don't currently do this for other peripherals, but I think we should.

Reply:
I was wondering the same! I thought there might be a reason for the fields not to be under `cfg` but only the `impl`?
If not I can do that change. Should I send another PR to apply `cfg` to all of the fields?

Reply:
> Should I send another PR to apply cfg to all of the fields?

I for one would like to see that.

Reply:
Just made #181 as per this discussion 🚀

Reply:
Will rebase on top of that one when (if) merged.

Reply:
I assume this comment is now resolved because of the decision that has been taken during the meeting referenced in #181?